src/tmuxp/workspace/builder.py

```diff
@@ -418,11 +429,21 @@ def get_pane_shell():
                     else:
                         return None
 
+                if has_gte_version("3.0"):
+                    environment = pconf.get("environment", wconf.get("environment"))
+                else:
+                    logging.warning(
+                        "Cannot set environment for new pane. "
+                        "You need tmux 3.0 or newer for this."
+                    )
+                    environment = {}
```

Comment:
I suppose we can remove the warning (and move it to libtmux perhaps?)

Reply:
Libtmux already has warnings in that case (see [here](https://github.com/tmux-python/libtmux/blob/6a03ad929368544cf07762799cdf4b1c821619ce/src/libtmux/session.py#L273) and [here](https://github.com/tmux-python/libtmux/blob/6a03ad929368544cf07762799cdf4b1c821619ce/src/libtmux/window.py#L532)). My idea is to (maybe in the future) provide a better warning (e.g. file name and line number etc.). Also, I wouldn't rely on a lib to provide warnings in such cases (besides that they probably also use different loggers, which might give different outputs).